include/class_loader/class_loader.hpp

```diff
@@ -130,6 +132,8 @@ class ClassLoader : public std::enable_shared_from_this<ClassLoader>
         std::bind(&ClassLoader::onPluginDeletion<Base>, shared_from_this(), std::placeholders::_1)
       );
     } catch (std::bad_weak_ptr & e) {  // This is not a shared_ptr
+      CONSOLE_BRIDGE_logWarn("class_loader::ClassLoader::createUniqueInstance "
+        "This class must be used with smart pointer");
```

Comment:
@ahcorde assuming `uncrustify` is happy with it, consider:
```suggestion
      CONSOLE_BRIDGE_logWarn(
        "To createUniqueInstance() with a class_loader::ClassLoader "
        "instance whose lifetime is not managed by an std::shared_ptr "
        "is deprecated.");
```
Same below.

Reply:
I still think the warning message could be improved. Also, can we emit it once?

Reply:
There is no API for this in `console_bridge` but I can add some logic, what do you think ?

Reply:
A simple `static bool` should do.